tests/common/mock_cgroup_commands.py

```diff
@@ -151,14 +173,34 @@ def mock_open(path, *args, **kwargs):
     def mock_path_exists(path):
         return original_path_exists(get_mapped_path(path))
 
+    data_files = [
+        (os.path.join(data_dir, 'init', 'walinuxagent.service'), UnitFilePaths.walinuxagent),
+        (os.path.join(data_dir, 'init', 'azure.slice'), UnitFilePaths.azure),
+        (os.path.join(data_dir, 'init', 'azure-vmextensions.slice'), UnitFilePaths.vmextensions)
+    ]
+
+    def add_data_file(source, target):
+        shutil.copyfile(source, get_mapped_path(target))
+
+    for items in data_files:
+        add_data_file(items[0], items[1])
+
     builtin_popen = "__builtin__.open" if sys.version_info[0] == 2 else "builtins.open"
-    with patch("azurelinuxagent.common.cgroupapi.subprocess.Popen", side_effect=mock_popen) as patcher:
+
+    with patch("azurelinuxagent.common.cgroupapi.subprocess.Popen", side_effect=mock_popen):
         with patch("azurelinuxagent.common.cgroupconfigurator.fileutil.mkdir", side_effect=mock_mkdir):
             with patch("azurelinuxagent.common.cgroupapi.os.path.exists", side_effect=mock_path_exists):
                 with patch(builtin_popen, side_effect=mock_open):
                     with patch('azurelinuxagent.common.cgroupapi.CGroupsApi.cgroups_supported', return_value=True):
                         with patch('azurelinuxagent.common.cgroupapi.CGroupsApi.is_systemd', return_value=True):
-                            patcher.__commands = __MOCKED_COMMANDS[:]
-                            patcher.add_command_mock = add_command_mock
-                            patcher.get_mapped_path = get_mapped_path
-                            yield patcher
+                            class Mock:
```

Comment:
Rename to `CgroupMock` maybe? Just to make it more clear

Reply:
It's gone in the next iteration of the mocks; will submit the PR shortly